Since 2.6.32 we need to include sched.h for TASK_INTERRUPTIBLE definition.

Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com.br>
---
 3rdparty/heci/heci_init.c |    1 +
 3rdparty/heci/heci_main.c |    1 +
 3rdparty/heci/io_heci.c   |    1 +
 3 files changed, 3 insertions(+)

diff -p -up linux-2.6.32/3rdparty/heci/heci_init.c.orig linux-2.6.32/3rdparty/heci/heci_init.c
--- linux-2.6.32/3rdparty/heci/heci_init.c.orig	2009-12-10 14:14:43.892174480 -0200
+++ linux-2.6.32/3rdparty/heci/heci_init.c	2009-12-10 14:14:55.715299673 -0200
@@ -48,6 +48,7 @@
 #include <linux/moduleparam.h>
 #include <linux/wait.h>
 #include <linux/delay.h>
+#include <linux/sched.h>
 
 #include "kcompat.h"
 #include "heci_data_structures.h"
diff -p -up linux-2.6.32/3rdparty/heci/heci_main.c.orig linux-2.6.32/3rdparty/heci/heci_main.c
--- linux-2.6.32/3rdparty/heci/heci_main.c.orig	2009-12-10 14:16:10.277299485 -0200
+++ linux-2.6.32/3rdparty/heci/heci_main.c	2009-12-10 14:16:23.646176666 -0200
@@ -57,6 +57,7 @@
 #include <linux/cdev.h>
 #include <asm/uaccess.h>
 #include <asm/unistd.h>
+#include <linux/sched.h>
 
 #include "kcompat.h"
 #include "heci.h"
diff -p -up linux-2.6.32/3rdparty/heci/io_heci.c.orig linux-2.6.32/3rdparty/heci/io_heci.c
--- linux-2.6.32/3rdparty/heci/io_heci.c.orig	2009-12-10 14:15:10.024298850 -0200
+++ linux-2.6.32/3rdparty/heci/io_heci.c	2009-12-10 14:15:24.544299599 -0200
@@ -58,6 +58,7 @@
 #include <asm/uaccess.h>
 #include <asm/unistd.h>
 #include <linux/delay.h>
+#include <linux/sched.h>
 
 #include "heci_data_structures.h"
 #include "heci.h"
